internal/provider/resource_flink_statement.go

```diff
@@ -273,6 +275,10 @@ func flinkStatementUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 	if oldStopped.(bool) == true && newStopped.(bool) == false {
 		return flinkStatementResume(ctx, d, meta)
 	}
+	_, sensitive, _ := extractFlinkProperties(d)
```

Comment:
it should be moved to `create()`: https://github.com/confluentinc/terraform-provider-confluent/blob/master/internal/provider/resource_connector.go#L159-L162

Reply:
this is already being saved in create. We need to add this in update to avoid terraform drift in the case of update to update to this